### ISSUE TITLE
Implement chunk generation server-side

### ISSUE THREAD

**Issue by zesterer** [veloren/game#57]

**Comment by treenix** [comment]
#32 will be relevant if we need to send chunks from the server to the clients.
Also the server should cache quite a lot of chunks instead of regenerating them all the time.

**Comment by zesterer** [comment]
@treenix Agreed. This needs further discussion. @xMAC94x's message-splitting system *should* be mature enough to do this, I think?
